en/docs/learn/metadata-cache-management.md

```diff
@@ -0,0 +1,144 @@
+Data Holders are required to be able to react to Data Recipient and associated software product status changes within
```

Comment:
Shall we reword this to:
`Data Holders must be able to respond to changes in the status of Data Recipients and associated software products on the CDR Register within 2-5 minutes of the occurrence of the change.`